src/content/changes/highlights/2024-09-26.md

```diff
@@ -0,0 +1,125 @@
+    - Allow configuring `kube-controller-manager` `--node-cidr-mask-size` flag.
+    - Chart: Support multiple service account issuers.\ Change `providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuer` to plural `providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers` and render them in the specified order as `--service-account-issuer` parameters for the API server.
+    - Only add the `customNodeLabels` value to the kubelet `node-labels` argument in the `KubeadmConfig` when `customNodeLabels` is defined.
+
+- [cluster-aws](https://github.com/giantswarm/cluster-aws) version [2.2.0](https://github.com/giantswarm/cluster-aws/compare/v2.1.0...v2.2.0)
+    - Allow to enable `auditd` through `global.components.auditd.enabled` helm value.
+    - Chart: Support multiple service account issuers.\ This is used for example in the migration from Vintage AWS clusters to CAPA. Multiple issuers were previously supported only through internal chart values (this change removes `internal.migration.irsaAdditionalDomain`). The internal annotation `aws.giantswarm.io/irsa-additional-domain` on AWSMachineTemplate objects is changed to plural `aws.giantswarm.io/irsa-trust-domains` on the AWSCluster object.
+    - Chart: Update `cluster` to v1.4.1.
+    - Set provider specific configuration for cilium CNI ENI values.
+    - Remove Cilium app deprecated values.
+
+- [cluster-azure](https://github.com/giantswarm/cluster-azure) version [1.3.0](https://github.com/giantswarm/cluster-azure/compare/v1.2.0...v1.3.0)
+    - Chart: Update `cluster` to [v1.4.1](https://github.com/giantswarm/cluster/releases/tag/v1.4.1)
+    - Allow to enable auditd service through `global.components.auditd.enabled`.
+    - Allow configuring `kube-controller-manager` `--node-cidr-mask-size` flag.
+    - Remove Cilium deprecated values.
+
+- [cluster-api-provider-cloud-director-app](https://github.com/giantswarm/cluster-api-provider-cloud-director-app) version [0.8.2](https://github.com/giantswarm/cluster-api-provider-cloud-director-app/compare/v0.8.1...v0.8.2)
+    - Use `gzip+base64` instead of `base64` for bootstrap data delivery to VM metadata.
+
+- [cluster-vsphere](https://github.com/giantswarm/cluster-vsphere) version [0.64.0](https://github.com/giantswarm/cluster-vsphere/compare/v0.63.0...v0.64.0)
+    - Migrated all worker resources (`KubeadmConfigTemplate`, `MachineDeployment` and `MachineHealthCheck`) to be rendered from the shared `cluster` chart.
+    - Render cleanup hook job using cluster chart.
+
+## Security
+
+- [kyverno-policies-observability](https://github.com/giantswarm/kyverno-policies-observability) version [0.5.0](https://github.com/giantswarm/kyverno-policies-observability/compare/v0.4.0...v0.5.0)
+    - Remove policy for `ServiceMonitor` and `PodMonitor` relabelling schemas as we do not need the enforcement anymore.
```

Comment:
Not sure if this lands here automatically but this is an Atlas topic, not security